Documentation/contributing/development/dev_setup.rst

```diff
@@ -108,6 +108,9 @@ brought up by vagrant:
   set, it defaults to ``docker``.
 * ``VAGRANT_DEFAULT_PROVIDER={virtualbox \| libvirt \| ...}``
 * ``VM_SET_PROXY=https://127.0.0.1:80/`` Sets up VM's ``https_proxy``.
+* ``INSTALL``: Restarts the installation of Cilium, Kubernetes, etc. Only
```

Comment:
Nit: all other variables are given a value (`=<something>`) in the doc, maybe do the same for `INSTALL`?

Also, should we mention explicitly that it is related to `RELOAD` (and overridden if the `RELOAD` is not set)?

Reply:
> Nit: all other variables are given a value (`=<something>`) in the doc, maybe do the same for `INSTALL`?

Oh, yes. That's an oversight.

> Also, should we mention explicitly that it is related to `RELOAD` (and overridden if the `RELOAD` is not set)?

It's not really overridden if `RELOAD` isn't set. `INSTALL` is set if `RELOAD` isn't set, but if `RELOAD` is set, it doesn't impact `INSTALL`.

Also note that `RELOAD` is automatically set by `set_reload_if_vm_exists()` so I'm not sure anyone actually manually sets `RELOAD` anymore...